### PR TITLE
Terjemahan BNPCHS 2018 - Final: Susu Moli

### PR DESCRIPTION
**Pull request by stevennovaryo**

### CODE REVIEW

File: translations/bnpchs-final-2018-Susu-Moli.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!-- Title: Moli Milk-->
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  Jelix likes recently popular street food called "Es Kepal Moli" a lot. After receiving Jelix's Es Kepal Moli, one of his friends, Fingga got curious, about the method of making Es Kepal Moli. Surprisingly one of the ingredients used by Jelix is Moli branded milk.
```
Comment (Berted) [inline translations/bnpchs-final-2018-Susu-Moli.html]:
```suggestion
  Jelix likes the newly popular street food called <em>Es Kepal Moli</em> a lot. 
  After receiving Jelix's <em>Es Kepal Moli</em>, one of his friends, Fingga got curious
  about the method of making Es Kepal Moli. 
  Surprisingly, one of the ingredients used by Jelix is Moli branded milk. 
```

File: translations/bnpchs-final-2018-Susu-Moli.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<!-- Title: Moli Milk-->
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  Jelix likes recently popular street food called "Es Kepal Moli" a lot. After receiving Jelix's Es Kepal Moli, one of his friends, Fingga got curious, about the method of making Es Kepal Moli. Surprisingly one of the ingredients used by Jelix is Moli branded milk. 
+</p>
+
+<p style="text-align: justify;">
+  Currently, there're $$$N$$$ Moli Milks in Jelix's fridge. Everyday Jelix will use $$$A$$$ Moli Milks to make Es Kepal Moli. To not run out of stock, Jelix buys $$$B$$$ Moli Milks every day and then store in the fridge. Surely, Jelix buys Moli Milk more than he uses each day. Fingga got curious after $$$K$$$ days, how many Moli Milks are in Jelix's fridge. Help Fingga to resolve it!
```
Comment (Berted) [inline translations/bnpchs-final-2018-Susu-Moli.html]:
```suggestion
  Currently, there are $$$N$$$ Moli Milk bottles in Jelix's fridge. 
  Everyday, Jelix will use $$$A$$$ Moli Milk bottles to make <em>Es Kepal Moli</em>. 
  To not run out of stock, Jelix also buys $$$B$$$ Moli Milk bottles every day and then stores them in the fridge. 
  Obviously, Jelix buys more Moli Milk bottles than the amount he uses each day. 
  Fingga is curious about how many Moli Milks there are in Jelix's fridge after $$$K$$$ days. 
  Help Fingga resolve it! 
```

File: translations/bnpchs-final-2018-Susu-Moli.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+</p>
+
+<p style="text-align: justify;">
+  Currently, there're $$$N$$$ Moli Milks in Jelix's fridge. Everyday Jelix will use $$$A$$$ Moli Milks to make Es Kepal Moli. To not run out of stock, Jelix buys $$$B$$$ Moli Milks every day and then store in the fridge. Surely, Jelix buys Moli Milk more than he uses each day. Fingga got curious after $$$K$$$ days, how many Moli Milks are in Jelix's fridge. Help Fingga to resolve it! 
+</p>
+
+<h3>Input Format</h3>
+
+<p style="text-align: justify;">
+  Input starts with a single integer $$$T (1 \leq T \leq 100)$$$ the number of test cases. The next lines contain 4 integer $$$N$$$, $$$A$$$, $$$B$$$, and $$$K$$$ $$$(1 \leq A < B, N \leq 100; 1 \leq K \leq 100)$$$.
```
Comment (Berted) [inline translations/bnpchs-final-2018-Susu-Moli.html]:
```suggestion
  Input starts with a single integer $$$T (1 \leq T \leq 100)$$$, the number of test cases. 
  The next lines each contains 4 integers $$$N$$$, $$$A$$$, $$$B$$$, and $$$K$$$ $$$(1 \leq A < B, N \leq 100; 1 \leq K \leq 100)$$$. 
```

File: translations/bnpchs-final-2018-Susu-Moli.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+<h3>Input Format</h3>
+
+<p style="text-align: justify;">
+  Input starts with a single integer $$$T (1 \leq T \leq 100)$$$ the number of test cases. The next lines contain 4 integer $$$N$$$, $$$A$$$, $$$B$$$, and $$$K$$$ $$$(1 \leq A < B, N \leq 100; 1 \leq K \leq 100)$$$. 
+</p>
+
+<h3>Output Format</h3>
+
+<p style="text-align: justify;">
+  For each test case, output "Kasus #$$$X$$$: $$$Y$$$", where $$$X$$$ is the case number (starting from 1), and $$$Y$$$ is the amount of milk in the fridge after $$$K$$$ days.
```
Comment (Berted) [inline translations/bnpchs-final-2018-Susu-Moli.html]:
```suggestion
  For each test case, output "Kasus #$$$X$$$: $$$Y$$$", 
  where $$$X$$$ is the case number (starting from 1), 
  and $$$Y$$$ is the number of milk bottles in the fridge after $$$K$$$ days.
```

**Review by Berted — CHANGES_REQUESTED** [review]
Multi-line makes it easier to review uwu

File: translations/bnpchs-final-2018-Susu-Moli.html (hunk 1 above)
Comment (stevennovaryo) [reply translations/bnpchs-final-2018-Susu-Moli.html]: uwu i used word wrap in vscode, always forgot to multi line

File: translations/bnpchs-final-2018-Susu-Moli.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  Jelix likes the newly popular street food called <em>Es Kepal Moli</em> a lot. 
+  After receiving Jelix's <em>Es Kepal Moli</em>, one of his friends, Fingga got curious
+  about the method of making <em>Es Kepal Moli</em>. 
+  Surprisingly, one of the ingredients used by Jelix is Moli branded milk.
+</p>
+
+<p style="text-align: justify;">
+  Currently, there are $$$N$$$ bottles of Moli Milk in Jelix's fridge. 
+  Everyday, Jelix will use $$$A$$$ Moli Milk bottles to make <em>Es Kepal Moli</em>. 
+  To not run out of stock, Jelix also buys $$$B$$$ Moli Milk bottles every day and then stores them in the fridge.
```
Comment (prabowo02) [inline translations/bnpchs-final-2018-Susu-Moli.html]:
```suggestion
  To keep in stock, Jelix also buys $$$B$$$ Moli Milk bottles every day and then stores them in the fridge. 
```

File: translations/bnpchs-final-2018-Susu-Moli.html [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  Obviously, Jelix buys more Moli Milk bottles than the amount he uses each day. 
+  Fingga is curious about how many bottles of Moli Milks there are in Jelix's fridge after $$$K$$$ days. 
+  Help Fingga resolve it!
+</p>
+
+<h3>Input Format</h3>
+
+<p style="text-align: justify;">
+  Input starts with a single integer $$$T (1 \leq T \leq 100)$$$ the number of test cases. 
+  The next lines contain 4 integers $$$N$$$, $$$A$$$, $$$B$$$, and $$$K$$$ $$$(1 \leq A < B, N \leq 100; 1 \leq K \leq 100)$$$.
```
Comment (prabowo02) [inline translations/bnpchs-final-2018-Susu-Moli.html]:
```suggestion
  Each case contains 4 integers $$$N$$$, $$$A$$$, $$$B$$$, and $$$K$$$ $$$(1 \leq A < B, N \leq 100; 1 \leq K \leq 100)$$$. 
```

**Review by prabowo02 — CHANGES_REQUESTED** [review]

**Review by Berted — APPROVED** [review]

**Review by prabowo02 — APPROVED** [review]

**Pull request merged**